resources/cloudwatchevents-buses.go

```diff
@@ -0,0 +1,50 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchevents"
+)
+
+func init() {
+	register("CloudWatchEventsBuses", ListCloudWatchEventsBuses)
+}
+
+func ListCloudWatchEventsBuses(sess *session.Session) ([]Resource, error) {
+	svc := cloudwatchevents.New(sess)
+
+	resp, err := svc.ListEventBuses(nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resources := make([]Resource, 0)
+	for _, bus := range resp.EventBuses {
+		if *bus.Name == "default" {
+			continue
+		}
+
+		resources = append(resources, &CloudWatchEventsBus{
+			svc:  svc,
+			name: bus.Name,
+		})
+	}
+	return resources, nil
+}
+
+type CloudWatchEventsBus struct {
+	svc  *cloudwatchevents.CloudWatchEvents
+	name *string
+}
+
+func (bus *CloudWatchEventsBus) Remove() error {
+	_, err := bus.svc.DeleteEventBus(&cloudwatchevents.DeleteEventBusInput{
+		Name: bus.name,
+	})
+	return err
+}
+
+func (bus *CloudWatchEventsBus) String() string {
+	return fmt.Sprintf("Bus: %s", *bus.name)
```

Comment:
```suggestion
	return *bus.name
```

Please just use the plain bus name, since `String()` is used for filters and having to prepend `Bus: ` might be confusing to users.

Reply:
@svenwltr Thank you for the feedback.  I added a second commit that returns the plain bus name.